### PR TITLE
AIDA-723 and AIDA-709

### PR DESCRIPTION
**Pull request by gitnordt**
Changed N/A thumbnails to display images instead on text, made lots of changes to the TA2 and TA3. All changes needed for the upcoming demo.

### CODE REVIEW

File: src/app/components/thumbnail-grid/thumbnail-grid.component.ts [hunk 1]
```diff
@@ -774,10 +774,25 @@ export class ThumbnailGridComponent extends BaseNeonComponent implements OnInit,
                     break;
                 }
                 default : {
-                    // todo: get thumbnails of documents, pdf, and other similar types of media.
-                    thumbnail.fillStyle = '#111111';
-                    thumbnail.font = '20px Helvetica Neue';
-                    thumbnail.fillText(fileType.toUpperCase(), 10, 30);
+                    if (link === 'n/a') {
+                        let img: HTMLImageElement = new Image();
+                        img.src = './assets/icons/document_viewer.svg';
```
Comment (ThomasSchellenbergNextCentury) [inline src/app/components/thumbnail-grid/thumbnail-grid.component.ts]: Would it make sense to show the document icon whenever the `link` or `fileType` is an empty value (like the empty string or `null`)?

**Review by ThomasSchellenbergNextCentury — DISMISSED** [review]

**Comment by ThomasSchellenbergNextCentury** [comment]
@snorpet Would you mind posting a screenshot just so I can see what it looks like?  Thanks :)

**Comment by gitnordt** [comment]
@ThomasSchellenbergNextCentury 
Screenshot from 2019-04-08 16-44-00

File: src/app/components/thumbnail-grid/thumbnail-grid.component.ts (hunk 1 above)
Comment (gitnordt) [reply src/app/components/thumbnail-grid/thumbnail-grid.component.ts]: Done, thank you for the suggestion.

**Review by ThomasSchellenbergNextCentury — APPROVED** [review]

**Pull request merged**